src/modules/menu/components/advancedTools/ImportFile/utils.js

```diff
@@ -50,8 +55,17 @@ export function handleFileContent(store, content, source) {
     } else if (isGpx(content)) {
         const gpxParser = new GPX()
         const metadata = gpxParser.readMetadata(content)
+        const parseGpx = new DOMParser().parseFromString(content, 'text/xml')
         layer = new GPXLayer(source, true, 1.0, content, metadata)
-        // TODO : zoom to extent
+        const extent = bbox(gpxToGeoJSON(parseGpx))
+        if (!extent) {
+            throw new EmptyGPXError()
```

Comment:
You also need to handle this error here https://github.com/geoadmin/web-mapviewer/blob/d9e436d7925739b78e7f3feee678ee6303da1841/src/modules/menu/components/advancedTools/ImportFile/ImportFileLocalTab.vue#L79